integration_tests/e2e/create-goal.cy.ts

```diff
@@ -19,4 +24,32 @@ describe('Create a new Goal', () => {
     cy.get('button').contains('Save and continue').click()
     cy.url().should('include', '/summary')
   })
+
+  it('Creates a new goal with errors', () => {
+    cy.url().should('include', '/about-pop')
+    cy.get("a[href='/create-goal/accommodation']").click()
+    cy.url().should('include', '/create-goal/accommodation')
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('#goal-name').type('acc').type('{downArrow},{enter}')
```

Comment:
now this isn't using keydown you can merge the two `type` command to `.type('acc{downArrow},{enter}')` and remove the eslint config - same for the two other instances of this below

Reply:
updated but lint config still required as its still complaining

Reply:
Fixed in [3ebd114](https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/61/commits/3ebd1144be53791fc316757045e82b901a66b633)